include/envoy/stats/stats.h

```diff
@@ -16,6 +16,7 @@ namespace Stats {
 
 class Allocator;
 struct Tag;
+using TagVector = std::vector<Tag>;
```

Comment:
can we can we just include tag.h here rather than predeclaring Tag and repeating the `using`?